common/app/assets/assets.scala

```diff
@@ -86,6 +86,7 @@ class Assets(base: String, assetMapPath: String = "assets/assets.map") extends L
        val knownInlines : PartialFunction[String,String] =
        {
          case "story-package" => "story-package.css"
+         case "KeepItInTheGround" => "basher.KeepItInTheGround.css"
```

Comment:
It would be less magic (and more explicit) if this just mapped `"basher.KeepItInTheGround" => "basher.KeepItInTheGround.css"`.